.circleci/config.yml

```diff
@@ -1,52 +1,65 @@
-# Python CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
 version: 2
 jobs:
-  build:
-    machine: true
-    #support this after we can enable it on account
-    #docker_layer_caching: true
+  test_centos7:
+    machine:
+      image: circleci/classic:latest
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - run:
+          name: Running tests CentOS 7
+          command: make test_centos7
+      - store_artifacts:
+          path: test-results
+      - store_test_results:
+          path: test-results
+  test_debian9:
+    machine:
+      image: circleci/classic:latest
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - run:
+          name: Running tests Debian 9 
+          command: make test_debian9
+      - store_artifacts:
+          path: test-results
+      - store_test_results:
+          path: test-results
+  security_vulnerability_scan:
+    machine:
+      image: circleci/classic:latest
+      docker_layer_caching: true
     working_directory: ~/repo
-
     steps:
-      # this section isn't needed for machine executors
-      #- setup_remote_docker
-           # enable this once the account supports it
-           #docker_layer_caching: true
```

Comment:
I'll check and see if we're getting this in our plan, if so this would mean you don't need to do a "docker save / load" and can just reuse the build cache.